analysis_options.yaml

```diff
@@ -1,9 +1,8 @@
 include: package:pedantic/analysis_options.yaml
 
 analyzer:
-  strong-mode:
-    implicit-casts: false
-    implicit-dynamic: false
```

Comment:
These two lints are not subsumed by null safety. You might want to keep them.
(Implicit dynamic is not changed, except that *some* types now defaults to `Object?`, and `implicit-casts` will still prevent implicit downcast from `dynamic`, which the language allows)